src/cli.ts

```diff
@@ -25,19 +25,9 @@ import {merge} from './merge-prs.js';
 import {main as check} from './repo-check.js';
 import {sync, exec} from './sync.js';
 import meow from 'meow';
-import updateNotifier from 'update-notifier';
 import {tag} from './tag-prs.js';
 import {untag} from './untag-prs.js';
-import * as fs from 'fs';
 import * as path from 'path';
```

Comment:
Looks like this could also be removed now?

Reply:
Indeed! Removed.